### DISCUSSION TITLE
Mapbox v10 swift compiler IOS build errors?

### DISCUSSION THREAD

**Discussion by xenvi** [rnmapbox/maps#2382] [Q&A]
Hello, I have been working on upgrading a project to mapbox v10 but have not been able to get the IOS side to build without errors. I followed the README-v10. I am getting swift compiler errors on MapboxMaps pod. Wondering if there is anything obvious I am missing? I have been stuck on this for some time now and have done numerous google searches and looked through issues/PRs in this repo but did not see anything similar.

@rnmapbox/maps = ^10.0.0-beta.54
Platform = iOS
React Native Version = 0.70.4
Device = any

Podfile:
`
...
$RNMapboxMapsImpl = 'mapbox'
platform :ios, '13'
...
 pre_install do |installer|
    $RNMapboxMaps.pre_install(installer)
    ... other pre install hooks
  end
...
 post_install do |installer|
    $RNMapboxMaps. post_install(installer)
    ... other post install hooks
  end
`

<img width="278" alt="Screen Shot 2022-11-04 at 5 13 12 PM" src="https://user-images.githubusercontent.com/49381727/200096139-8355fc20-e1f6-4b3d-8017-35fd642f63f8.png">

**Comment by mfazekas** [comment]
Interesting because empty is there.  Can you check GeoJSONSourceData if it has `case empty`?!

https://github.com/mapbox/mapbox-maps-ios/blob/7055ad31cb3d6d38ed38ae465fe7aca8a1aace15/Sources/MapboxMaps/Style/Types/GeoJSONSourceData.swift#L19-L20

**Reply by xenvi** [reply]
Thank you for the reply!

I have checked the Pod and there is a case empty. I did a text compare for that GeoJSONSourceData.swift file in my pod compared to the repo code and it is an exact match
